packages.config

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="OSPSuite.SimModelSolver_CVODES" version="4.1.0.8" />
-  <package id="OSPSuite.SimModelSolver_CVODES.Ubuntu18" version="4.1.0.8" />
+  <package id="OSPSuite.SimModelSolver_CVODES" version="4.1.0.9" />
+  <package id="OSPSuite.SimModelSolver_CVODES.Ubuntu22" version="4.1.0.9" />
   <package id="OSPSuite.SimModelSolver_CVODES.CentOS7" version="4.1.0.8" />
```

Comment:
Didn't create a new version of CentOS-packages. 
I assume we will remove them anyways later and provide only Ubuntu version.